src/ast/passes/codegen_llvm.cpp

```diff
@@ -49,6 +49,8 @@
 
 namespace bpftrace::ast {
 
+using Function = llvm::Function;
```

Comment:
super nit (non blocking) but I might rather just adding the namespace to all instances of `Function` (like you do in some of the other files below) to distinguish at first glance. Same for the other file where you add a `using`